.github/workflows/daily.yaml

```diff
@@ -0,0 +1,61 @@
+name: Daily Merge and CodeSonar run
+on:
+  push:
+    branches:
+      - release/daily
+  pull_request:
+    branches:
+      - 'release/daily'
+  schedule:
+    - cron: '17 2 * * *'
+
+jobs:
+  Daily_Merge_and_CodeSonar_Analyze:
+    permissions: write-all
+    runs-on: self-hosted
+    container:
+      image: canuckmh/wolfssl-cso-builder:7.4p0
+      credentials:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+    env:
+      CAFILE: ".github/github.cert.pem"
+      TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      API_URL: https://api.github.com
+      REQUEST_NUMBER: ${{ github.event.pull_request.number }}
+      ROOT_TREE: "WolfSSL-Projects/WolfSSL-GitHub"
+      PROJECT_NAME: "wolfssl"
+      BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+      COMMIT_HASH: $GITHUB_SHA
+      CSONAR_HUB_URL: "https://partnerdemo.codesonar.com"
+      CSONAR_HUB_USER: "${{ secrets.CSONAR_HUB_USER }}"
+      CSONAR_HUB_PASSWORD: "${{ secrets.CSONAR_HUB_PASS }}"
+      CSONAR_CSHOME: /opt/codesonar
+      CSO_GITHUB: /opt/codesonar-github
+      REPO_URL: "http://github.com/markhermeling/wolfssl"    # the github.repositoryUrl is not an http-link.
```

Comment:
With this setup would this test have a dependency on the forked wolfssl repository?

Reply:
It does not. It merges from daily-snapshot into my own fork and runs the analysis there. I figured the separation would be nice.